### ISSUE TITLE
# symbol misinterpreted as comment when in string

### ISSUE THREAD

**Issue by DanieleBenedettelli** [openmv/openmv#551]
### Steps to reproduce
1. type this code in a module (not main program)
```
line = '#something'
if (b'#' in line): # discard comments
	print('comment!')
```

2. Tools->Save open script to cam (send module too when asked)

3. Run program

### Expected behaviour
The code should work ok.

### Actual behaviour
IDE complains, because the code in the module became:
```
	line = '
	if (b'
		print('comment!')
```

The # symbol was interpreted as comment, even if it was not at the beginning of the line.

### Describe your setup

**IDE version:**
IDE 2.2.0

**Firmware version:**
MicroPython v1.9.4-4553-gb4eccdfe3 on 2019-05-02; OPENMV4 with STM32H743

**Operating system:**
Win 10
### For developers

**GCC version:**

**Build target:**

**Comment by kwagyeman** [comment]
Yes, this is a known issue. It will be fixed in the next IDE release. I am back to programming work and we are doing software updates again .
